### ISSUE TITLE
Excalidraw react component should support react server components, aka nextjs 13 without "use client"

### ISSUE THREAD

**Issue by NateRadebaugh** [excalidraw/excalidraw#6170]

**Comment by ad1992** [comment]
@NateRadebaugh could you elaborate? In `next` you can disable [ssr](https://nextjs.org/docs/advanced-features/dynamic-import#with-no-ssr) for components not supporting server side rendering.

**Comment by NateRadebaugh** [comment]
Yes I think what I'm asking for is to explicitly support SSR. For instance, I have a use case where the server generates a static chart that I'd like to be rendered to the front-end, ideally without any javascript at all since I don't need it to be interactive.

**Comment by dwelle** [comment]
@NateRadebaugh your use case could be covered by a server-side renderer (`excalidraw (json)` -> `png`/`svg`), but that's only when the drawing is small. For large drawings you still need interaction to pan around and support zooming.
